keyboards/inett_studio/sq80/info.json

```diff
@@ -0,0 +1,227 @@
+                { "matrix": [3, 13], "x": 17.25, "y": 1.5 },
+
+                { "matrix": [5, 1], "w": 1.5, "x": 0, "y": 2.5 },
+                { "matrix": [2, 0], "x": 1.5, "y": 2.5 },
+                { "matrix": [2, 1], "x": 2.5, "y": 2.5 },
+                { "matrix": [2, 2], "x": 3.5, "y": 2.5 },
+                { "matrix": [2, 3], "x": 4.5, "y": 2.5 },
+                { "matrix": [2, 4], "x": 5.5, "y": 2.5 },
+                { "matrix": [2, 5], "x": 6.5, "y": 2.5 },
+                { "matrix": [2, 6], "x": 7.5, "y": 2.5 },
+                { "matrix": [2, 7], "x": 8.5, "y": 2.5 },
+                { "matrix": [2, 8], "x": 9.5, "y": 2.5 },
+                { "matrix": [2, 9], "x": 10.5, "y": 2.5 },
+                { "matrix": [2, 10], "x": 11.5, "y": 2.5 },
+                { "matrix": [2, 11], "x": 12.5, "y": 2.5 },
+                { "matrix": [2, 12], "w": 1.5, "x": 13.5, "y": 2.5 },
+                { "matrix": [2, 13], "x": 15.25, "y": 2.5 },
+                { "matrix": [2, 14], "x": 16.25, "y": 2.5 },
+                { "matrix": [4, 14], "x": 17.25, "y": 2.5 },
+
+                { "matrix": [5, 2], "w": 1.75, "x": 0, "y": 3.5 },
+                { "matrix": [3, 0], "x": 1.75, "y": 3.5 },
+                { "matrix": [3, 1], "x": 2.75, "y": 3.5 },
+                { "matrix": [3, 2], "x": 3.75, "y": 3.5 },
+                { "matrix": [3, 3], "x": 4.75, "y": 3.5 },
+                { "matrix": [3, 4], "x": 5.75, "y": 3.5 },
+                { "matrix": [3, 5], "x": 6.75, "y": 3.5 },
+                { "matrix": [3, 6], "x": 7.75, "y": 3.5 },
+                { "matrix": [3, 7], "x": 8.75, "y": 3.5 },
+                { "matrix": [3, 8], "x": 9.75, "y": 3.5 },
+                { "matrix": [3, 9], "x": 10.75, "y": 3.5 },
+                { "matrix": [3, 10], "x": 11.75, "y": 3.5 },
+                { "matrix": [3, 11], "w": 2.25, "x": 12.75, "y": 3.5 },
+
+                { "matrix": [5, 3], "w": 2.25, "x": 0, "y": 4.5 },
+                { "matrix": [4, 0], "x": 2.25, "y": 4.5 },
+                { "matrix": [4, 1], "x": 3.25, "y": 4.5 },
+                { "matrix": [4, 2], "x": 4.25, "y": 4.5 },
+                { "matrix": [4, 3], "x": 5.25, "y": 4.5 },
+                { "matrix": [4, 4], "x": 6.25, "y": 4.5 },
+                { "matrix": [4, 5], "x": 7.25, "y": 4.5 },
+                { "matrix": [4, 6], "x": 8.25, "y": 4.5 },
+                { "matrix": [4, 7], "x": 9.25, "y": 4.5 },
+                { "matrix": [4, 8], "x": 10.25, "y": 4.5 },
+                { "matrix": [4, 9], "x": 11.25, "y": 4.5 },
+                { "matrix": [4, 10], "w": 2.75, "x": 12.25, "y": 4.5 },
+                { "matrix": [4, 13], "x": 16.25, "y": 4.5 },
+
+                { "matrix": [5, 4], "w": 1.5, "x": 0, "y": 5.5 },
+                { "matrix": [5, 6], "w": 1.5, "x": 2.5, "y": 5.5 },
+                { "matrix": [5, 7], "w": 7, "x": 4, "y": 5.5 },
+                { "matrix": [5, 8], "w": 1.5, "x": 11, "y": 5.5 },
```

Comment:
quick check: Is matrix co-ordinate for this key (1.5u key right side of 7u spacebar) `[5, 8]` or `[5, 9]`?